### DISCUSSION TITLE
Bump go-redis to v9?

### DISCUSSION THREAD

**Discussion by mfelipe** [testcontainers/testcontainers-go#2700] [Q&A] [answered]
Just wanted to ask... are there any reason to stick with [go-redis](https://github.com/redis/go-redis) v8 instead of v9 for the redis module?

Cheers!

**Comment by mdelapenya** [comment, marked as answer]
Hi @mfelipe which issues do you see in that version? We are open to bumping the version, just want to check the motivation for the changes.

Thanks!

**Reply by mfelipe** [reply]
Hi @mdelapenya, I currently have no issues regarding v8, but I was thinking about the bugfixes and features that v9 to introduces.

**But** that was a short-sight from my side since it is just used for Ping, Set and Get testing... whichever version is used is fine, I guess. :)

**Reply by mdelapenya** [reply]
In any case I encourage you to submit a PR to the module if you're still interested in doing that.

**Reply by mdelapenya** [reply]
> it is just used for Ping, Set and Get testing.

Indeed. We try not to couple the modules to any given client technology, so that developers choose their own flavour. We of course need to test the modules, so we use a client only for the tests in the module.
